static/js/stream_popover.js

```diff
@@ -406,12 +406,24 @@ function build_move_topic_to_stream_popover(e, current_stream_id, topic_name) {
 
     hide_topic_popover();
 
-    function move_topic() {
-        const params = Object.fromEntries(
+    function get_params_from_form() {
+        return Object.fromEntries(
             $("#move_topic_form")
                 .serializeArray()
                 .map(({name, value}) => [name, value]),
         );
+    }
+
+    function allow_or_disallow_submission(select_stream_id) {
```

Comment:
I think a better name might be `update_submit_button_disabled_state` ; this name is ambiguous for whether it's a function called when you click the submit button or what it actually does.

Reply:
Noted, will incorporate the change.